.github/ISSUE_TEMPLATE/-maintainer--new-feature.md

```diff
@@ -0,0 +1,20 @@
+---
+name: "(Maintainer) New Feature"
+about: New feature description, filed by CRM maintainers
+title: ''
+labels: enhancement, source::maintainer
+assignees: ''
+
```

Comment:
Is there a way we can restrict this option? Like can this only be shown for maintainers with some config flag?

Reply:
No, GH does offer the ability to restrict access to issue templates to select users